quartz.config.ts

```diff
@@ -72,7 +73,7 @@ const config: QuartzConfig = {
     filters: [Plugin.RemoveDrafts()],
     emitters: [
       Plugin.AliasRedirects(),
-      Plugin.ComponentResources({ fontOrigin: "googleFonts" }),
+      Plugin.ComponentResources(),
       Plugin.ContentPage(),
       Plugin.FolderPage(),
       Plugin.TagPage(),
```

Comment:
this might cause breakages when merging for people

Reply:
Yeah, its unfortunate, but I think its better to move the font origin to theme.
- cdnCaching is also a prop on theme and fontOrigin is related
- lets us access fontOrigin in Head.tsx, so we can conditionally inject html to fetch from google fonts